src/main/jhipster/src/main/webapp/app/release-notes/release-notes.component.ts

```diff
@@ -0,0 +1,116 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { AlertService } from '../shared/alert/alert.service';
+import { ActivatedRoute } from '@angular/router';
+import { PopupService } from '../shared/modal/popup.service';
+import { NgbActiveModal } from '@ng-bootstrap/ng-bootstrap';
+import { JhiLanguageService } from 'ng-jhipster';
+import { ReleaseNotesService } from './release-notes.service';
+import { HttpResponse } from '@angular/common/http';
+import { ReleaseNote } from './release-notes.model';
+import { SafeResourceUrl } from '@angular/platform-browser/src/security/dom_sanitization_service';
+import { DomSanitizer } from '@angular/platform-browser';
+import { ReleaseNoteContext } from './release-note.context';
+
+@Component({
+    selector: 'jhi-release-notes-dialog',
+    templateUrl: './release-notes.component.html',
+    styleUrls: [
+        'release-notes.scss'
+    ],
+})
+export class ReleaseNotesDialogComponent implements OnInit {
+
+    dontShowAgain = true;
```

Comment:
dontShowAgain by default seem kind of weird. The user: "I like to see this popup next time, so I will uncheck this to make sure that happens"

Reply:
Done!

Reply:
dontShowAgain will not be checked by default. Agreed with @mcrimi